test/specs/commonTests/implementsShorthandProp.tsx

```diff
@@ -3,10 +3,12 @@ import { mount } from './isConformant'
 
 export type ShorthandTestOptions = {
   mapsValueToProp?: string
+  skipArrayOfStrings?: boolean
 }
 
```

Comment:
Let's not change this interface, but create new one specific for the collection shorthand, as skipArrayOfStrings is not in any way used in the simple shorthand. The last option how the shorthand can be used is a function with signature: `(Component, props, children) => React.Node`. If we are not supporting this one, maybe we should add config for this one as well.